src/renderer/components/TimelineSpace/Contents/Cards/Toot.vue

```diff
@@ -316,6 +330,9 @@ export default {
     },
     isShowAttachments (message) {
       return !this.sensitive(message) || this.showAttachments
+    },
+    filterd (message) {
```

Comment:
__[Misspell]__

"filterd" is a misspelling of "filtered"


<!-- Comment by [Sider](https://sider.review) -->